src/services/files/upload-file.service.ts

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+import { AxiosError } from "axios";
+import { ENVIRONMENT } from "../../config/environment";
+
+type UploadfileRequest = {
+  fileContent: Uint8Array;
+  fileName: string;
+  location: string;
+  token: string;
+};
+
+type UploadfileResponse = {
+  msg: string;
+  success: boolean;
+  fileUUID?: string;
+};
+
+export const uploadfileService = async (
+  req: UploadfileRequest
+): Promise<UploadfileResponse> => {
+  try {
+    const response = await axios.post(
+      `${ENVIRONMENT.PROXY_BASE_URL}/file/upload`,
+      req,
```

Comment:
The upload endpoint doesn't expect a JSON. You should send the file via `multipart/form-data`, no encoding needed. There is an example using `Insomnia`: 

![image](https://github.com/hawks-atlanta/frontend-react/assets/62714297/113ba9f4-0678-49f3-afd5-b95ae675f723)

Reply:
Then i send the file as a File type?
`type UploadfileRequest = {
  fileContent: File;
  fileName: string;
  location: string;
  token: string;
};`

Reply:
Yes.

Reply:
I am trying to upload the files but the response says there are missing fields.
![image](https://github.com/hawks-atlanta/frontend-react/assets/131629166/da5a9ade-6ef8-4e7a-a954-a95eb3f9d0e9)